R/pkg/R/generics.R

```diff
@@ -421,6 +421,10 @@ setGeneric("corr", function(x, ...) {standardGeneric("corr") })
 #' @export
 setGeneric("describe", function(x, col, ...) { standardGeneric("describe") })
 
+#' @rdname drop
+#' @export
+setGeneric("drop", function(x, col) { standardGeneric("drop") })
```

Comment:
does base::drop work? https://stat.ethz.ch/R-manual/R-devel/library/base/html/drop.html
if yes could you add a test?